packages/nextjs/src/pages/__tests__/index.test.tsx

```diff
@@ -31,27 +31,6 @@ describe('ClerkProvider', () => {
     it('domain + isSatellite (satellite app)', () => {
       expectTypeOf({ ...defaultProps, domain: 'test', isSatellite: true }).toMatchTypeOf<ClerkProviderProps>();
     });
-
-    it('only domain is not allowed', () => {
-      expectTypeOf({ ...defaultProps, domain: 'test' }).not.toMatchTypeOf<ClerkProviderProps>();
-    });
-
-    it('only isSatellite is not allowed', () => {
-      expectTypeOf({ ...defaultProps, isSatellite: true }).not.toMatchTypeOf<ClerkProviderProps>();
-    });
-
-    it('proxyUrl + domain is not allowed', () => {
-      expectTypeOf({ ...defaultProps, proxyUrl: 'test', domain: 'test' }).not.toMatchTypeOf<ClerkProviderProps>();
-    });
-
-    it('proxyUrl + domain + isSatellite is not allowed', () => {
-      expectTypeOf({
-        ...defaultProps,
-        proxyUrl: 'test',
-        domain: 'test',
-        isSatellite: true,
-      }).not.toMatchTypeOf<ClerkProviderProps>();
-    });
```

Comment:
❓ Did we loosen up the type of `MultiDomainProxy`?

Reply:
In my opinion, these types greatly increased our type system's complexity without providing the benefit I'd expect.

I'd like to discuss with @panteliselef more about this before merging as he has much more context than me :)